assets/js/atomic/blocks/product-elements/tag-list/index.js

```diff
@@ -14,13 +14,18 @@ import {
 	BLOCK_ICON as icon,
 	BLOCK_DESCRIPTION as description,
 } from './constants';
+import { Save } from './save';
+import { supports } from './supports';
```

Comment:
I'm not sure why should we use a separated file instead of using `index.ts`?

Reply:
Just a stylistic choice. I preferred to keep the `index.js` short